index.js

```diff
@@ -53,6 +53,7 @@ module.exports = {
     'mocha/no-pending-tests': 'error',
     'mocha/no-skipped-tests': 'warn',
     'mocha/valid-test-description': 'error',
+    'no-template-curly-in-string': 'off',
```

Comment:
Any repo that uses bunsen. Can have their bunsen models reference object like `id: '${value}',`. Thus eslint will error in those consuming repos like `mcp-ui`